neqo-common/src/lib.rs

```diff
@@ -9,6 +9,8 @@
 mod codec;
 mod datagram;
 pub mod event;
+#[cfg(feature = "build-fuzzing-corpus")]
+pub mod fuzz;
```

Comment:
Because you are re-exporting this...
```suggestion
mod fuzz;
```